hack/shellcheck-test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+  
+# Check for shell syntax & style.
+
```

Comment:
You need to set SHELLCHECK somewhere initially, like so:

```suggestion

SHELLCHECK="$(command -v shellcheck 2>/dev/null)"
```

Otherwise no shellcheck will actually be run...